omero/sysadmins/whatsnew.txt

```diff
@@ -7,7 +7,17 @@ What's new for OMERO 5.3 for sysadmins
   the Ice Python bindings if your package manager does not provide the Ice
   Python packages.
 
-- All OMERO web apps can now be installed using ``pip install``.
+- :doc:`version-requirements` has been updated to reflect other changes in
+  version support for 5.3.0 and tentative plans for 5.4.0.
+  
+- All official OMERO web apps can now be installed using ``pip install``. If
+  you have been running Django 1.6.x prior to upgrading, you should remove and
+  reinstall your plugins when you upgrade.
```

Comment:
yes. this will still be necessary to unzip for some e.g. webtagging is not pip installable
but all the official omero ones are now available from pypi.